web/src/classic/components/molecules/Dashboard/QuickStart.tsx

```diff
@@ -46,11 +47,19 @@ const QuickStart: React.FC<Props> = ({
   const [prjectType, setPrjectType] = useState<ProjectType>("classic");
   const theme = useTheme();
   const isSmallWindow = useMedia("(max-width: 1024px)");
+  const { useMeQuery } = useMeFetcher();
+  const {
+    me: { email },
+  } = useMeQuery();
 
   const handleCreateProjectClick = useCallback(() => {
-    if (window.REEARTH_CONFIG?.developerMode) setPrjTypeSelectOpen(true);
+    if (
+      email &&
```

Comment:
checking if developerMode shouldn't care about email, so keep email connected with the superAdmin check only